config/initializers/newspaer.rb

```diff
@@ -6,6 +6,7 @@
   Newspaper.subscribe(:answer_create, NotifierToWatchingUser.new)
   Newspaper.subscribe(:announcement_destroy, AnnouncementNotificationDestroyer.new)
   Newspaper.subscribe(:answer_create, AnswererWatcher.new)
+  Newspaper.subscribe(:announcement_create, AnnouncementNotifier.new)
```

Comment:
🙏ご提案まで🙏
あくまでご提案まで〜という感じなのですが、FBCアプリ内での通知とdiscordでの通知に関するコードが、元は別々のメソッド(`send_notification`と`notify_to_chat`)で分けられていたので

```suggestion
  Newspaper.subscribe(:announcement_create, AnnouncementNotifier.new)
  Newspaper.subscribe(:announcement_create, AnnouncementChatNotifier.new)
```
こんな感じで（もっといい命名があるかもしれないんですが💦）、置き換え後も別のクラスに分けた方がコードが何をしているか分かりやすいかな？と感じたことを、共有しておきたいと思います〜✨！

Reply:
確かにこちらの方が処理の粒度が細かくなるので良いですね！ご提案ありがとうございます🙏